<launch>

  <group ns = "No_0">
    <node pkg = "trailing_mpc" type = "virtual_base" name = "virtual_base" output = "screen">  

    </node>

    <node pkg = "trailing_mpc" type = "virtual_environment" name = "virtual_environment" output = "screen">  

    </node>
  </group>

  <group ns = "No_1">

    <node pkg = "trailing_mpc" type = "virtual_base" name = "virtual_base" output = "screen">  

    </node>
        <node pkg = "trailing_mpc" type = "virtual_environment" name = "virtual_environment" output = "screen">  

    </node>

  </group>

    <group ns = "No_2">

    <node pkg = "trailing_mpc" type = "virtual_base" name = "virtual_base" output = "screen">  

    </node>
        <node pkg = "trailing_mpc" type = "virtual_environment" name = "virtual_environment" output = "screen">  

    </node>

  </group>

  <group ns = "No_4">

    <node pkg = "trailing_mpc" type = "virtual_base" name = "virtual_base" output = "screen">  

    </node>
        <node pkg = "trailing_mpc" type = "virtual_environment" name = "virtual_environment" output = "screen">  

    </node>

  </group>

</launch>